### PR TITLE
Update file_info.go (fix errcheck issue)

### PR DESCRIPTION
**Pull request by TheInvincibleRalph**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28926

```release-note
NONE
```

### CODE REVIEW

File: server/channels/app/file_info.go [hunk 1]
```diff
@@ -52,6 +55,14 @@ func getInfoForBytes(name string, data io.ReadSeeker, size int) (*model.FileInfo
...
 			}
 		}
+	} else {
+		return nil, model.NewAppError(
+			"getInfoForBytes",
+			"app.file_info.decode_image.app_error",
+			nil,
+			err.Error(),
+			http.StatusUnsupportedMediaType,
+		).Wrap(err)
```
Comment (hanzei) [inline server/channels/app/file_info.go:65]: This seems wrong. Which error are you wrapping here?

File: server/channels/app/file_info.go [hunk 2]
```diff
@@ -40,7 +41,9 @@ func getInfoForBytes(name string, data io.ReadSeeker, size int) (*model.FileInfo
 
 			if info.MimeType == "image/gif" {
 				// Just show the gif itself instead of a preview image for animated gifs
-				data.Seek(0, io.SeekStart)
+				if _, err := data.Seek(0, io.SeekStart); err != nil {
+					log.Printf("Failed to seek to the beginning of the data: %v", err)
```
Comment (hanzei) [inline server/channels/app/file_info.go:45]: Let's not log to stdout and instead return an `*model.AppError`. Maybe there is an error code that you can reuse.

**Review by hanzei — CHANGES_REQUESTED** [review]
Another one!

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Review by gabrieljackson — APPROVED** [review]
LGTM

**Comment by hanzei** [comment]
@TheInvincibleRalph Did you do anything fancy with the branch? I can neither merge the PR nor push to your branch.

**Comment by TheInvincibleRalph** [comment]
> @TheInvincibleRalph Did you do anything fancy with the branch? I can neither merge the PR nor push to your branch.

No. The branch is still intact.

**Pull request merged**